install_files/ansible-base/roles/install_fpf_repo/tasks/main.yml

```diff
@@ -3,9 +3,15 @@
   apt_key: >
```

Comment:
I don't really understand why you're relying on Ansible to actually add the key into the trusted keyring or why this approach is superior to the previous work that I did. All your package does is put it on the filesystem and help with rotations.

I recommend that you add an [install](https://www.debian.org/doc/manuals/maint-guide/dother.en.html#install) file so it's clear that you have files destined for `/etc/apt/trusted.gpg.d`.